src/quacc/recipes/emt/phonons.py

```diff
@@ -27,6 +27,7 @@ def phonon_flow(
         tuple[tuple[int, int, int], tuple[int, int, int], tuple[int, int, int]] | None
     ) = None,
     displacement: float = 0.01,
+    additional_atoms: Atoms | None = None,
```

Comment:
Note to self: we'll want to add this parameter to the other recipes calling the `phonon_subflow` (tblite and MLP I think?).